srcopsmetrics/kebechet_sli_slo_metrics.py

```diff
@@ -60,6 +60,22 @@ def _get_usage_sli_advise_manager(self, kebechet_metrics):
             "is_used": is_used,
         }
 
+    def _get_source_code_changes_sli(self, kebechet_metrics, bot_prs):
+        human_prs = kebechet_metrics._get_human_pull_request(filter_file="Pipfile.lock")
```

Comment:
Just a conversation starter here from my side. Having underscore function with prefix `_` would mean that the function should be private and no one outside the class / file should access it (that's my understanding it, I can be wrong here of course).

So, if this function is implemented in kebechet_metrics.py class and is used here, it should have name without underscore, meaning it's not intended to be used only privately within class 

I mean, it's just really a minor detail and there are plenty of other functions already that violate this rule, so my question is if this is the right way of using it. And if so, we can start using it now and perhaps look at the other aesthetic erros in the code later. WDYT @Gkrumbach07 ?

Reply:
> should be private and no one outside the class

Yes I agree, I think I also violated that too. I can go through and update the underscores.

Reply:
As for other aesthetic changes, we should try to stay consistent in how we create pr/issue dataframes for each manager. I saw you had some filter function, but i dont know if this is being used everywhere.